scripts/geany-release.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import shutil
+from os.path import exists, isfile, join
+from subprocess import check_call
+
+"""
+This script prepares a Geany release on Windows.
+The following steps will be executed:
+- update geany-themes repository
+- strip binary files (geany.exe, plugin .dlls)
+- sign binary files with certificate
+- create installers
+- sign installers
+"""
+
+VERSION = '2.1'
+# adjust paths to your needs ($HOME is used because expanduser() returns the Windows home directory)
+BASE_DIR = join(os.environ['HOME'], 'geany_build')
+SOURCE_DIR = join(os.environ['HOME'], 'git', 'geany')
+BUILD_DIR = join(os.environ['HOME'], 'git', 'geany', '_build')
+RELEASE_DIR_ORIG = join(BASE_DIR, 'release', 'geany-orig')
+RELEASE_DIR = join(BASE_DIR, 'release', 'geany')
+BUNDLE_BASE_DIR = join(BASE_DIR, 'bundle')
+BUNDLE_GTK = join(BASE_DIR, 'bundle', 'geany-gtk')
+GEANY_THEMES_URL = 'https://github.com/geany/geany-themes/archive/refs/heads/master.zip'
+GEANY_THEMES_DIR = join(BUNDLE_BASE_DIR, 'geany-themes')
+INSTALLER_NAME = join(BASE_DIR, f'geany-{VERSION}_setup.exe')
+
+# signing params
+SIGN_CERTIFICATE = join(BASE_DIR, 'codesign.pem')  # adjust to your needs
+SIGN_CERTIFICATE_KEY = join(BASE_DIR, 'codesign_key.pem')  # adjust to your needs
+SIGN_WEBSITE = 'https://www.geany.org'
+SIGN_NAME = 'Geany Binary'
+SIGN_TIMESTAMP = 'https://zeitstempel.dfn.de/'
```

Comment:
After a bit of research this looks probably OK, but without further explanation it looks *so* shady it'd be good to explain it.  @eht16 can probably help here(?)

Reply:
I explained parts of the signing topic in https://github.com/geany/geany/issues/4189#issuecomment-2629001461.

The short story is, we will probably not sign the binaries any longer and so we could also remove the related code.

Reply:
@giuspen since we will not use this kind of code signing any longer, feel free to remove the relevant code.